front/lib/api/tables.ts

```diff
@@ -24,7 +33,8 @@ type CsvParsingError = {
     | "duplicate_header"
     | "invalid_record_length"
     | "empty_csv"
-    | "invalid_row_id";
+    | "invalid_row_id"
+    | "app_error";
```

Comment:
app error is IMO not super clear (you need to know that there's an app and why)

Reply:
right, actually we don't get into this case anymore as I fallback to the previous method if the app fails ..